core_api/tests/conftest.py

```diff
@@ -95,8 +94,8 @@ def stored_file_1(elasticsearch_storage_handler, file) -> File:
 
 
 @pytest.fixture()
-def embedding_model_dim(embedding_model) -> int:
-    return len(embedding_model.embed_query("foo"))
+def embedding_model_dim() -> int:
+    return 1536 #text-embedding-3-small default dimension
```

Comment:
For text-embedding-3-large the dimension size is 3072

https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#embeddings-models

Reply:
do we want to use text-embedding-3-large in development? It would be cleaner but I'm thinking about costs? We can use ada-002 in our region is it worth doing that?

Reply:
We're all good for cost concerns for now, going to use 3-large everywhere